app/serializers/api/order_detailed_serializer.rb

```diff
@@ -0,0 +1,12 @@
+module Api
+  class OrderDetailedSerializer < Api::Admin::OrderSerializer
+    has_one :shipping_method, serializer: Api::ShippingMethodSerializer
+    has_one :ship_address, serializer: Api::AddressSerializer
+    has_one :bill_address, serializer: Api::AddressSerializer
+
+    has_many :line_items, serializer: Api::LineItemSerializer
+    has_many :adjustments, serializer: Api::AdjustmentSerializer
```

Comment:
⚠️ see my comment above

Reply:
I think we are ok in terms of adjustments, there's one query for them.